src/package.xml

```diff
@@ -203,8 +203,6 @@
         <members>Account.HEDA_Account_Compact_Layout</members>
         <members>Address__c.HEDA_Address_Compact_Layout</members>
         <members>Affiliation__c.HEDA_Affiliation_Compact_Layout</members>
-        <members>Behavior_Involvement__c.EDA_Behavior_Involvement_Compact_Layout</members>
```

Comment:
This was done automatically... I think maybe they were previously removed but didn't get removed from the package.xml?